Documents/Pseudocode/Pseudocode_Final

```diff
@@ -152,26 +152,26 @@ Class Network
 	
 	//takes a matrix object and the address to a function (returns a double, no arguments)
 	//assigns numbers as specified by the function passed
-	void randomizeMatrix(matrix<double> input_matrix, double (*distribution)(void))
+	void randomizeMatrix(matrix<double> input_matrix, double (*distribution)(void)) //I am still thinking about distribution(), need to look up for that [-Ami]
```

Comment:
I'm new to functors and the like, so whatever you find will probably serve us better!